Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -379,7 +379,6 @@ namespace AZ
         {
             if (m_systemAssetsInitialized)
             {
-                AZ_Warning("RPISystem", false , "InitializeSystemAssets should only be called once'");
```

Comment:
Can I ask why is it removed? Delaying the initialization doesn't seem to affect the warning.

Reply:
I think it's okay to have the function be called more than once and it shouldn't be warning since it's handled gracefully. 
Currently the material editor manually call this function manually and it's earlier than the queued function RPISystemComponent did. so it would be called twice. and there is no harm the function is called twice.